### ISSUE TITLE
Speed up the inference manager step on buildkite

### ISSUE THREAD

**Issue by paul-butcher** [wellcomecollection/catalogue-pipeline#2445]

**Comment by paul-butcher** [comment]
Actually, just harmonising the base would be a big speedup, as the Docker cache will be able to get past the installation of gcc et al.

However, harmonising the Python dependencies where possible would also be sensible

**Comment by paul-butcher** [comment]
Actually, there's something else here too.

The inference manager step builds these three containers, and each of the steps to test them individually also builds the one associated with itself.

**Comment by paul-butcher** [comment]
The reason the builds from the three individual inferrers cannot be reused for the inference manager is that we have no guarantee they are running on the same agent.  Everything happens in parallel.

Reusing specific agents for multiple stages in Buildkite is frowned upon.

**Comment by kenoir** [comment]
Closing as won't do given move from Buildkite to GHA.
